src/screens/TopicPageScreen/index.js

```diff
@@ -161,7 +161,7 @@ const TopicPageScreen = (props) => {
 
   const getTopicDetail = async (domain) => {
     try {
-      const resultTopicDetail = await getTopics(domain);
+      const resultTopicDetail = await getTopics(domain, false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `getTopics` function is now being called with two parameters, but the second parameter `false` is not clear in terms of what it represents. Consider using an object as a parameter to improve readability and clarity.

```diff
- const resultTopicDetail = await getTopics(domain, false);
+ const resultTopicDetail = await getTopics({ domain, withMinimumFollower: false });
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->